hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/RpcWritable.java

```diff
@@ -106,7 +106,7 @@ Message getMessage() {
     @Override
     void writeTo(ResponseBuffer out) throws IOException {
       int length = message.getSerializedSize();
-      length += CodedOutputStream.computeRawVarint32Size(length);
+      length += CodedOutputStream.computeUInt32SizeNoTag(length);
```

Comment:
Change from varint to 32bit? This breaks compatibility?

Reply:
`computeRawVarint32Size()` is deprecated and internally it calls `computeUInt32SizeNoTag()` itself. So used directly `computeUInt32SizeNoTag()` to avoid javac warnings.
I don't think this is an incompatible change.